docs/pages/components/checkbox.md

```diff
@@ -0,0 +1,199 @@
+	<label class="fd-checkbox__label" for="dOKph699">Label</label>
+  
+    <input type="checkbox" class="fd-checkbox is-warning" id="Fyp8j109" name="checkbox" indeterminate>
+	<label class="fd-checkbox__label" for="Fyp8j109">Label</label>
+  
+    <input type="checkbox" class="fd-checkbox is-information" id="h4z5N521" name="checkbox" indeterminate>
+	<label class="fd-checkbox__label" for="h4z5N521">Label</label>
+{% endcapture %}
+
+{% include display-component.html component=checkbox-interaction-states %}
+
+## Checkbox RTL mode
+
+{% capture checkbox-rtl %}
+    <div dir="RTL">
+        <input type="checkbox" class="fd-checkbox is-success" id="z3h5E577" name="checkbox" >
+        <label class="fd-checkbox__label" for="z3h5E577">Label</label>
+    
+        <input type="checkbox" class="fd-checkbox is-error" id="wzgmo573" name="checkbox" >
+        <label class="fd-checkbox__label" for="wzgmo573">Label</label>
+    
+        <input type="checkbox" class="fd-checkbox is-warning" id="xKsvq131" name="checkbox" >
+        <label class="fd-checkbox__label" for="xKsvq131">Label</label>
+    
+        <input type="checkbox" class="fd-checkbox is-information" id="oIwYW653" name="checkbox" >
+        <label class="fd-checkbox__label" for="oIwYW653">Label</label>
+    
+        <input type="checkbox" class="fd-checkbox is-success" id="BVVnS213" name="checkbox" checked>
+        <label class="fd-checkbox__label" for="BVVnS213">Label</label>
+    
+        <input type="checkbox" class="fd-checkbox is-error" id="o6jYz764" name="checkbox" checked>
+        <label class="fd-checkbox__label" for="o6jYz764">Label</label>
+    
+        <input type="checkbox" class="fd-checkbox is-warning" id="IcRL5321" name="checkbox" checked>
+        <label class="fd-checkbox__label" for="IcRL5321">Label</label>
+    
+        <input type="checkbox" class="fd-checkbox is-information" id="NrhH7399" name="checkbox" checked>
+        <label class="fd-checkbox__label" for="NrhH7399">Label</label>
+    
+        <input type="checkbox" class="fd-checkbox is-success" id="x1JgL105" name="checkbox" indeterminate>
+        <label class="fd-checkbox__label" for="x1JgL105">Label</label>
+    
+        <input type="checkbox" class="fd-checkbox is-error" id="dOKph699" name="checkbox" indeterminate>
+        <label class="fd-checkbox__label" for="dOKph699">Label</label>
+    
+        <input type="checkbox" class="fd-checkbox is-warning" id="Fyp8j109" name="checkbox" indeterminate>
+        <label class="fd-checkbox__label" for="Fyp8j109">Label</label>
+    
+        <input type="checkbox" class="fd-checkbox is-information" id="h4z5N521" name="checkbox" indeterminate>
+        <label class="fd-checkbox__label" for="h4z5N521">Label</label>
+    </div>
+{% endcapture %}
+
+{% include display-component.html component=checkbox-rtl %}
+
+## Display Only Checkbox
+
+{% capture checkbox-rtl %}
+    <input type="checkbox" class="fd-checkbox--display-only" id="1tn4O446" checked style="display:none">
```

Comment:
This should be `fd-checkbox fd-checkbox--display-only`. It should always have `fd-checkbox`